inspirehep/bat/pages/create_literature.py

```diff
@@ -247,9 +247,6 @@ def _chapter_info_population(input_data):
 
 def _links_population(input_data):
     Arsenic().find_element_by_id('url').send_keys(input_data['pdf-1'])
-    Arsenic().find_element_by_id('additional_url').send_keys(
-        input_data['pdf-2']
-    )
```

Comment:
The request made by this guy's validator was not being mocked in the `acceptance` tests... for now I removed it, but the real fix is to prepend a commit that mocks this as well in `wsgi_with_coverage`.